yt_dlp/options.py

```diff
@@ -408,6 +408,10 @@ def _alias_callback(option, opt_str, value, parser, opts, nargs):
         help=(
             'Location of the main configuration file; either the path to the config or its containing directory '
             '("-" for stdin). Can be used multiple times and inside other configuration files'))
+    general.add_option(
+        '--plugin-locations',
+        dest='plugin_locations', metavar='PATH', action='append',
```

Comment:
I think this should also sognify its internal state by having an underscore prefix
```suggestion
        dest='_plugin_locations', metavar='PATH', action='append',
```

Reply:
Actually, disregard this. If we plan on using the same flag (which we will), this variable and its value will not change and should be safe to consume.